apps/video/js/view.js

```diff
@@ -210,14 +218,9 @@ navigator.mozSetMessageHandler('activity', function viewVideo(activity) {
     }
     // We cannot change the visibility state of video contorls when we are in
     // picking mode.
-    if (!controlShowing) {
-      // If control not shown, tap any place to show it.
-      setControlsVisibility(true);
-      e.cancelBubble = true;
-    } else if (e.originalTarget === dom.videoControls) {
-      // If control is shown, only tap the empty area should show it.
-      setControlsVisibility(false);
-    }
+    e.cancelBubble = !controlShowing;
+    // If control not shown, tap any place to show it.
+    setControlsVisibility(!controlShowing);
```

Comment:
This comment is no longer relevant given the code change. My suggestion is to remove it.